bin/ci/run-tests.js

```diff
@@ -58,7 +58,21 @@ Object.keys(fileSizes).forEach(key => {
 });
 
 const otherTests = [
-  "browser_net_open_in_debugger.js"
+  "devtools/client/framework/test/browser_browser_toolbox_debugger.js",
+  "devtools/client/framework/test/browser_keybindings_01.js",
+  "devtools/client/framework/test/browser_source_map-01.js",
+  "devtools/client/framework/test/browser_source_map-absolute.js",
+  "devtools/client/framework/test/browser_source_map-init.js",
+  "devtools/client/framework/test/browser_source_map-inline.js",
```

Comment:
are the source-map tests all using the debugger?

Reply:
They need a toolbox and to do so they open it on the debugger. I assumed they needed to start the debugger for the tests to work. But starting the toolbox on any other tool seems to be sufficient to make the tests pass. I'll remove those.

Reply:
I'd rather not include them, because a smaller list is easier...

we can always re-consider if they fail in a release :P